packages/node-cli/src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -88,23 +88,26 @@ module.exports = class ImportObjectsCommandGenerator extends BaseCommandGenerato
 		if (Array.isArray(data) && listObjectsResult.data.length === 0) {
 			throw TranslationService.getMessage(MESSAGES.NO_OBJECTS_TO_LIST);
 		}
-		
+
 		let selectionObjectAnswers;
 		let anwersAfterObjectSelection;
+		let overwriteConfirmationAnswer;
 		try {
 			const selectionObjectQuestions = this._generateSelectionObjectQuestions(listObjectsResult);
 			selectionObjectAnswers = await prompt(selectionObjectQuestions);
 
 			const questionsAfterObjectSelection = this._generateQuestionsAfterObjectSelection(selectionObjectAnswers);
 			anwersAfterObjectSelection = await prompt(questionsAfterObjectSelection);
+
+			const overwriteConfirmationQuestion = this._generateOverwriteConfirmationQuestion(anwersAfterObjectSelection);
+			overwriteConfirmationAnswer = await prompt(overwriteConfirmationQuestion);
 		} catch (error) {
 			throw TranslationService.getMessage(PROMPTING_INTERACTIVE_QUESTIONS_FAILED, NodeUtils.lineBreak, error);
-		}			
+		}
 
-		const combinedAnswers = { ...listObjectAnswers, ...selectionObjectAnswers, ...anwersAfterObjectSelection };
+		const combinedAnswers = { ...listObjectAnswers, ...selectionObjectAnswers, ...anwersAfterObjectSelection, ...overwriteConfirmationAnswer };
```

Comment:
Maybe we should stop the execution here instead of passing the "overwriteConfirmationAnswer". But as is the first thing checked in _executeAction() I'm fine with it.